carbonmark-api/test/routes/listings/[id]/get.test.ts

```diff
@@ -0,0 +1,38 @@
+import { FastifyInstance } from "fastify";
+import nock from "nock";
+import { GRAPH_URLS } from "../../../../src/app.constants";
+import { fixtures } from "../../../fixtures";
+import { build } from "../../../helper";
+import { DEV_URL } from "../../../test.constants";
+
+describe("GET /listings/:id", () => {
+  let fastify: FastifyInstance;
+
+  // Setup the server
+  beforeEach(async () => {
+    try {
+      fastify = await build();
+    } catch (e) {
+      console.error("/listings/:id get.test.ts setup failed", e);
+    }
+  });
+  test("Returns listing with token symbol", async () => {
```

Comment:
also need tests for sad paths:
- 404 when no listing found
- 500 server error when the listing was found but no token was found

Reply:
Done 👍